src/app/Library/CrudPanel/Traits/HasViewNamespaces.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Traits;
+
+use Backpack\CRUD\ViewNamespaces;
+
+trait HasViewNamespaces
+{
+    /**
+     * This file is only needed because we messed up version constrains from 1.2 up to 1.2.6 of PRO version
+     * and any user that the license ended in the middle of those versions was not able to update
+     * Backpack/CRUD up from 5.3.6.
+     *
+     * This should be removed in the next major version.
```

Comment:
```suggestion
     * This file is only needed because we messed up version constrains from 
     * 1.2 up to 1.2.6 of PRO version and any user that the license ended 
     * in the middle of those versions was not able to update
     * Backpack/CRUD up from 5.3.6.
     *
     * This should be removed in the next major version.
```